### PR TITLE
Parallelize entry processing in replay stage in validators

### PR DESCRIPTION
**Pull request by pgarg66**
#### Problem
Single threaded entry processing is not utlizing CPU cores to the fullest. The replay stage is taking long to process the entry and submit vote afterwards. This results in higher confirmation time.

#### Summary of Changes
Use process_entries() instead of process_entry(). This functions tries to run as many entries in parallel as possible. Also there's a fix for a corner case where accounts were not getting unlocked.

### CODE REVIEW

File: src/bank.rs [hunk 1]
```diff
@@ -649,6 +649,15 @@ impl Bank {
                 self.par_execute_entries(&mt_group)?;
                 mt_group = vec![];
                 //reset the lock and push the entry
+                locked
+                    .iter()
+                    .zip(entry.transactions.iter())
+                    .for_each(|(result, tx)| {
+                        if result.is_ok() {
+                            println!("Unlocking accounts for tx");
```
Comment (aeyakovenko) [inline src/bank.rs]: s/println/trace

Comment (pgarg66) [reply src/bank.rs]: oops

File: src/bank.rs [hunk 2]
```diff
@@ -649,6 +649,15 @@ impl Bank {
                 self.par_execute_entries(&mt_group)?;
                 mt_group = vec![];
                 //reset the lock and push the entry
+                locked
```
Comment (rob-solana) [inline src/bank.rs]: I think you can just call unlock_accounts() directly...  unlock_account() checks the result, and unlocks iff the failure wasn't "failure to lock", i.e AccountInUse

Comment (pgarg66) [reply src/bank.rs]: 👍

**Pull request merged**